### PR TITLE
1816 Use profile locale

### PR DESCRIPTION
**Pull request by Daniel-Echegaray**

### CODE REVIEW

File: Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift [hunk 1]
```diff
@@ -194,7 +194,10 @@ public struct SimplifiedLoginViewControllerRepresentable: UIViewControllerRepres
             }
         }
         
-        let s = SimplifiedLoginUIFactory.buildViewController(client: client, env: .pre, withMFA: .password, loginHint: nil, extraScopeValues: [], withSSO: true, completion: completion) as! UINavigationController
+        let context = UserContextFromTokenResponse(identifier: "An identifier", display_text: "a display text", client_name: "A client name")
+        let profile = UserProfileResponse(locale: "sv_SE")
```
Comment (zamorastp) [inline Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift:198]: TODO :) shouldn't be hardcoded

**Comment by zamorastp** [comment]
In old SDK, standard iOS localization was done (.lproj): 
https://github.com/schibsted/account-sdk-ios/tree/master/Source/UI/Resources

To not avoid users and developers confusion  is good to keep the same way of localization

**Comment by Daniel-Echegaray** [comment]
> In old SDK, standard iOS localization was done (.lproj): https://github.com/schibsted/account-sdk-ios/tree/master/Source/UI/Resources
> 
> To not avoid users and developers confusion is good to keep the same way of localization

Didnt we  want the SimplifiedLogin view to be coherent with the language used in the webflow?

**Comment by zamorastp** [comment]
> > In old SDK, standard iOS localization was done (.lproj): https://github.com/schibsted/account-sdk-ios/tree/master/Source/UI/Resources
> > To not avoid users and developers confusion is good to keep the same way of localization
> 
> Didnt we want the SimplifiedLogin view to be coherent with the language used in the webflow?

Yes, but we were also talking about checking how it was done in the old sdk. Does the current solution address the situation when for example Finnish speaking person who lives in Sweden, uses a Swedish portal but wants to have Finnish language?? If we use locale from User Profile that everything's all right :) - there is no necessity switching to .lproj solution now

**Comment by Daniel-Echegaray** [comment]
> > > In old SDK, standard iOS localization was done (.lproj): https://github.com/schibsted/account-sdk-ios/tree/master/Source/UI/Resources
> > > To not avoid users and developers confusion is good to keep the same way of localization
> > 
> > 
> > Didnt we want the SimplifiedLogin view to be coherent with the language used in the webflow?
> 
> Yes, but we were also talking about checking how it was done in the old sdk. Does the current solution address the situation when for example Finnish speaking person who lives in Sweden, uses a Swedish portal but wants to have Finnish language?? If we use locale from User Profile that everything's all right :) - there is no necessity switching to .lproj solution now

The cases that you mention "Finnish speaking person who lives in Sweden, uses a Swedish portal but wants to have Finnish language". 
By only using the language identifier part of the locale we will avoid such cases.
The locale we get from the backend consists of a language identifier  (e.g. "sv") concatenated with "_" and thereafter region identifier (eg. "NO").

I created a ticket to look into the .Iproj solution https://github.schibsted.io/spt-identity/t/issues/1838

File: Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift (hunk 1 above)
Comment (Daniel-Echegaray) [reply Sources/AccountSDKIOSWeb/Simplified Login/SimplifiedLoginViewController.swift]: added it

**Pull request closed without merging**